src/redux/reducers/pageReducer.js

```diff
@@ -1,9 +1,8 @@
 const InitialState = {
```

Comment:
Looks like I missed it last time. By default convention PascalCase is used for class names, and common variables should be in camelCase.